apps/backend/src/services/device_service.py

```
@@ -21,7 +21,7 @@
     DeviceSummary, DeviceHealth, DeviceConnectionTest, DeviceMetricsOverview
 )
 from apps.backend.src.schemas.common import OperationResult, PaginationParams, DeviceStatus
-from apps.backend.src.utils.ssh_client import get_ssh_client, SSHConnectionInfo, test_ssh_connectivity
+from apps.backend.src.utils.ssh_client import get_ssh_client, SSHConnectionInfo, test_ssh_connectivity_simple
```

Ruff (0.12.2)

`apps.backend.src.utils.ssh_client.get_ssh_client` imported but unused

Remove unused import

(F401)